lib/vtls/vtls.c

```diff
@@ -248,7 +248,7 @@ Curl_ssl_connect(struct connectdata *conn, int sockindex)
   conn->ssl[sockindex].use = TRUE;
   conn->ssl[sockindex].state = ssl_connection_negotiating;
 
-  result = Curl_ssl->connect(conn, sockindex);
+  result = Curl_ssl->connect_blocking(conn, sockindex);
```

Comment:
This isn't gcc's fault, this is some silly headers you use that seems to implement some of the socket API functions as macros and thus banning the function names from use everywhere. I'd say that is pretty bad style but I also realize you're not to blame for that...

Reply:
Yes, it's the bsdsocket inline header.  All Amiga 68k libraries are called using macros like that, the macros set the values in the correct registers then jump to the function code.  Unfortunately not much I can do about that, it was a design decision taken ~35 years ago!